Client/src/bkr/client/commands/cmd_update_inventory.py

```diff
@@ -87,14 +87,14 @@
 
 from __future__ import print_function
 
-import cgi
 import sys
 from xml.dom.minidom import parseString
 
 from requests.exceptions import HTTPError
 
 from bkr.client import BeakerCommand
 from bkr.client.task_watcher import watch_tasks
+from bkr.common.helpers_py3 import parse_content_type
```

Comment:
`helpers_py3` will be used in python2 and python3 imports (as we are still building the client for Python 2). This is a bit misleading. I prefer to rename this file.

Reply:
Renamed it to `helpers_six` as it works with python2 and python3, when whole codebase will be python3 compatible and we drop python2 support, we could move everything to original file.

Reply:
Sounds good. Thanks!